client/post-editor/editor-post-formats/index.jsx

```diff
@@ -1,44 +1,42 @@
 /**
  * External dependencies
  */
-const React = require( 'react' );
+import React, { PropTypes } from 'react';
+import { map, some } from 'lodash';
 
 /**
  * Internal dependencies
  */
-const FormRadio = require( 'components/forms/form-radio' ),
-	Gridicon = require( 'components/gridicon' ),
-	PostActions = require( 'lib/posts/actions' ),
-	stats = require( 'lib/posts/stats' ),
-	AccordionSection = require( 'components/accordion/section' );
+import FormRadio from 'components/forms/form-radio';
+import Gridicon from 'components/gridicon';
+import PostActions from 'lib/posts/actions';
+import stats from 'lib/posts/stats';
```

Comment:
I noticed an error while testing.

<img width="1274" alt="capture d ecran 2016-10-10 a 2 44 11 pm" src="https://cloud.githubusercontent.com/assets/272444/19238280/c0265c52-8ef8-11e6-985e-3ef5b2715c83.png">

I think the fix is something like `import * as stats from 'lib/posts/stats'` here. or better import only the functions you need `recordEvent` ect..